Resources/Prototypes/_Omu/Entities/Clothing/Head/mantles.yml

```
@@ -117,6 +117,29 @@
       - state: eyes-engineering
         shader: unshaded
```

Engi mantle should be engi hardsuit equivalent too if we're giving that type of stuff for atmos.
```suggestion
        shader: unshaded
  - type: ComponentToggler
    components: # copied from the parent with some additions
    - type: VisionCorrection
      correctionPower: 16
    - type: NightVision
      isEquipment: true
      color: "#CCFFCC"
    - type: FlashImmunity
    - type: FlashSoundSuppression
    - type: BreathMask
    - type: PressureProtection # Different pressure resist taken from engi hardsuit helmet.
      highPressureMultiplier: 0.1 
      lowPressureMultiplier: 1000
    - type: EyeProtection # And also weld prot.
```